defaults.lua

```diff
@@ -72,9 +72,9 @@ FOLLOW_LINK_TIMEOUT = 0.5
 -- y: y coordinate of top left corner in proportion to screen height
 -- w: tap zone width in proportion to screen width
 -- h: tap zone height in proportion to screen height
-DTAP_ZONE_MENU = {x = 1/8, y = 0, w = 3/4, h = 1/8}
-DTAP_ZONE_CONFIG = {x = 1/8, y = 7/8, w = 3/4, h = 1/8}
-DTAP_ZONE_MINIBAR = {x = 0, y = 31/32, w = 1, h = 1/32}
+DTAP_ZONE_MENU = {x = 1/8, y = 0, w = 3/4, h = 1/3}
+DTAP_ZONE_CONFIG = {x = 1/8, y = 3/4, w = 3/4, h = 1/4}
+DTAP_ZONE_MINIBAR = {x = 0, y = 12/13, w = 1, h = 1/13}
```

Comment:
13 in 12/13 does not feel multiple of 2 like all the other do :) Dunno if that matters in any way, and if we math.floor/round all thar properly and if that matters :)

Reply:
it shouldn't, I'm guessing fractions were used to make the computations easier to grok (i.e., y+h == 1).

I'm actually using `h = 0.075` & y = `0.925` here, and this was the closest.

Reply:
Getting to those exact values using fractions would mean using `12.333/13.333` and `1/13.333` which is *definitely* worse ^^.